[DM-1320] Update documentation for connectivity provider (#642)
SoftwareAG/c8y-docs: content/users-guide/device-management-bundle/connectivity.md

1. Click **Connectivity** in the **Settings** menu of the navigator. If the menu item is not displayed, make sure that your user has [READ and ADMIN permissions for Connectivity](/users-guide/administration#managing-permissions). If the menu item is still not available, please contact [product support](/welcome/contacting-support/) to make the Connectivity agent available in your tenant.
2. Switch to the **SIM provider settings** tab.
3. Select a provider from the drop-down list.
1. Enter the credentials for the respective SIM provider account. If you do not have any credentials, ask your administrator.
2. Click **Save** to save your settings.

> Depending on the connectivity provider you choose you would need to provide parameters which are specific for every provider.  **Cache duration** defines how long SIM data returned by the provider is cached before fresh data is requested again, in seconds. This mechnism aims at reducing traffic and number of requests to SIM providers for billing and reliability purposes. Longer cache duration means less traffic to your SIM provider, shorter duration means more current data is displayed.

Reviewer: ```suggestion
Depending on the connectivity provider you choose you must provide parameters specific to the provider.
The parameter **Cache duration** defines how long SIM data returned by the provider is cached before fresh data is requested again, in seconds.
This reduces traffic and the number of requests to SIM providers for billing and reliability purposes.
A longer cache duration means less traffic to your SIM provider while a shorter duration means that more recent data is displayed.
```
not 100% sure about current -> recent @bozhidardedov

Author: To me it looks OK, @l2p- ?

Reviewer 2: I think since points 2,3, and 4 all refer to the cache duration parameter they should not be separate points an the same level. They can be combined to a single point or somehow grouped separately.

Reviewer: this will render as a paragraph, it's just more readable in the source file.
